papers/Infrastructure_Project_Contact_List.md

```diff
@@ -0,0 +1,37 @@
+# RAKU INFRASTRUCTURE PROJECT LIST
+
+This paper is intended to hold a list of Raku infrastructures projects where
+maintainers are looking for volunteers to join them in their duties.
+
+## Adding A Project To The List
+
+To add a project to this list a maintainer must open a PR with necessary
+addition to this paper. A new project entry must be added under the _PROJECT
+LIST_ section and contain the following elements:
+
+1. Project title. It must be a second-level markdown heading like: _## IRC Bot
+   For Something_
+2. Brief project description.
+3. Perhaps a few words on what kind of help is required.
+4. Maintainer contact information. This could be anything that would allow a
+   volunteer to get in touch with you: an email (though don't forget to take
+   necessary measures to complicate spammers life), and IRC handle, a social
```

Comment:
"an" IRC handle

Reply:
Thanks, both corrections addressed in edd54d0